compiler/src/dotty/tools/dotc/printing/DecompilerPrinter.scala

```diff
@@ -29,8 +36,10 @@ class DecompilerPrinter(_ctx: Context) extends RefinedPrinter(_ctx) {
       case _ => toTextGlobal(stats, "\n")
     }
     val bodyText =
-      if (currentPrecedence == TopLevelPrec) "\n" ~ statsText else " {" ~ statsText ~ "}"
-    keywordStr("package ") ~ toTextPackageId(tree.pid) ~ bodyText
+      if (tree.pid.symbol.isEmptyPackage) statsText
```

Comment:
Isn't this case redundant since you discard it on line 42 if  `tree.pid.symbol.isEmptyPackage`?

Reply:
The body is the only part that is not discarded